doc/renaming.xml

```diff
@@ -0,0 +1,7 @@
+<Chapter Label="Renaming">
+<Heading>Renaming</Heading>
+
+In this chapter a table of renamed methods is included.
+The old names are explained in <Cite Key="IssacRecog"/>.
+
+<#Include SYSTEM "renaming_table.xml">
```

Comment:
We call auto generated files `_NAME.xml` that is they start with an underscore. Files inside `doc/` of this form are ignored by git per our `.gitignore` file.